lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -48,6 +49,7 @@ import { JWT } from '../utils/jwt';
  * @typedef ContentInfoLinks
  * @prop {string} [previousItem] - Previous item in the book, journal etc.
  * @prop {string} [nextItem] - Next item in the book, journal etc.
+ * @prop {string} currentItem
```

Comment:
Adding in a generated link to the current item so we can link to it in the banner.